tuist/tuist
# Lint target references in a scheme

Resolves <https://github.com/tuist/tuist/issues/6467>

### Short description 📝

> Describe here the purpose of your PR.
Resolve 6467 and check targetReferences in Workspace

### How to test the changes locally 🧐

You can check the [project](https://github.com/finebel/TuistSampleSchemeCreation) and the [issue](https://github.com/tuist/tuist/issues/6467)

> Include a set of steps for the reviewer to test the changes locally (see [the documentation](https://docs.tuist.io/contributors/get-started) for reference).

### Contributor checklist ✅

- [ ✅] The code has been linted using run `mise run lint-fix`
- [ ✅ ] The change is tested via unit testing or acceptance testing, or both
- [ ✅] The title of the PR is formulated in a way that is usable as a changelog entry
- [ ✅] In case the PR introduces changes that affect users, the documentation has been updated

### Reviewer checklist ✅
